src/content/about.mdx

```diff
@@ -6,29 +6,31 @@ title: Homepage
 import DownloadFile from '../components/download/DownloadFile.jsx'
 import DocumentLink from '../components/link/DocumentLink.jsx'
 import Notification from '@zlab-de/zel-react/Notification'
+import List from "@zlab-de/zel-react/List"
+import ListItem from '@zlab-de/zel-react/ListItem'
 
-# Zeppelin Design System
 
-This Design System enables you to build rich enterprise experiences and custom applications with the patterns and established best practices that are native to Zeppelin.
+<h1 style={{textAlign: 'center'}}> Zeppelin Design System </h1>
 
-<Notification variant="danger">
-  This website contains documentation for version 1 of the Zeppelin Element
-  Library
+<p style={{textAlign: 'center'}}>An online living style guide for Zeppelin Gmbh </p>
```

Comment:
It's not just for Zeppelin GmbH! 
Should be "An online living style guide for Zeppelin Group"